backend/capellacollab/core/models.py

```diff
@@ -9,10 +9,18 @@
 
 
 class Message(pydantic.BaseModel):
-    err_code: str | None = None
-    title: str | None = None
-    reason: str | tuple | None = None
-    technical: str | None = None
+    err_code: str = pydantic.Field(
+        description="The error code of the message, used for testing, not displayed in the frontend.",
+        examples=["T4C_SERVER_UNREACHABLE"],
+    )
+    title: str = pydantic.Field(
+        description="The title of the message, displayed in the frontend",
+        examples=["TeamForCapella server not reachable"],
+    )
+    reason: str = pydantic.Field(
```

Comment:
Since you've dropped support for tuple, we can also adjust the frontend implementation for it. Have a look at the `error-handling.interceptor.ts`.

The parts `Array.isArray(error.reason)` and `Array.isArray(warning.reason)` can be removed.